cves/CVE-2012-3439.yml

```diff
@@ -30,7 +30,7 @@ announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
```

Comment:
I give this 2 points for upvotes. It seems alarming how a whole authentication can be bypassed with just a randomly assigned number.

Reply:
I've addressed most of your comments. Thank you for all the feedback. It was very helpful.